build-system/tasks/e2e/functional-test-controller.js

```diff
@@ -315,6 +324,15 @@ class FunctionalTestController {
    */
   async isElementEnabled(unusedHandle) {}
 
+  /**
+   * Return an array of handles consisting of all windows in the browser
+   * session.
+   * {@link https://www.w3.org/TR/webdriver1/#get-window-handles}
+   *
+   * @return {!Promise<Array<string>>}
+   */
+  async getAllWindows() {}
```

Comment:
nit: move this method next to `switchToWIndow`

Reply:
Currently functional-test-controller places all getters together and all switchTo functions together, so this would adhere to their convention.